src/main/java/com/ibm/devops/dra/AbstractDevOpsAction.java

```diff
@@ -111,24 +110,24 @@
     );
 
     private static Map<String, String> POLICIES_URL_MAP = ImmutableMap.of(
-            "production", "https://dra.ng.bluemix.net/api/v4/organizations/{org_name}/toolchainids/{toolchain_name}/policies",
-            "dev", "https://dev-dra.stage1.ng.bluemix.net/api/v4/organizations/{org_name}/toolchainids/{toolchain_name}/policies",
-            "new", "https://new-dra.stage1.ng.bluemix.net/api/v4/organizations/{org_name}/toolchainids/{toolchain_name}/policies",
```

Comment:
We have remove the "new" deployment both in gate service and dlms. you probably want to remove it from here as well. The only deployment we have is "dev", "stage" and "prod"